src/main/java/io/github/thebusybiscuit/exoticgarden/items/FoodRegistry.java

```diff
@@ -1,531 +1,617 @@
+        18)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("FRIED_CHICKEN", Material.COOKED_CHICKEN, "&eFried Chicken", "&7&o\"Want some Grease? Eat some Fried Chicken!\" - @Wencrow", "", "&7&oRestores &b&o" + "3.5" + " &7&oHunger"),
+        new ItemStack[] {new ItemStack(Material.CHICKEN), new ItemStack(Material.EGG), SlimefunItems.WHEAT_FLOUR, SlimefunItems.SALT, getItem("BLACK_PEPPER"), getItem("VEGETABLE_OIL"), null, null, null},
+        7)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("FRIED_CHICKEN_BUCKET", "b8fb0502a3aa5f8bd32a5ea5e519c3dd353234170dfef959ee8adb9487fea", "&eFried Chicken Bucket", "", "&7&oRestores &b&o" + "21.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), null, new ItemStack(Material.BUCKET), null},
+        42)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("FRIED_CHICKEN_SANDWICH", "f7b9f08ada4e8ba586a04ed2e9e25fe8b9d568a665243f9c603799a7c896736", "&eFried Chicken Sandwich", "", "&7&oRestores &b&o" + "5.5" + " &7&oHunger"),
+        new ItemStack[] {new ItemStack(Material.BREAD), getItem("FRIED_CHICKEN"), null, null, null, null, null, null, null},
+        11)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("FRIED_CHICKEN_AND_CHEESE_SANDWICH", "f7b9f08ada4e8ba586a04ed2e9e25fe8b9d568a665243f9c603799a7c896736", "&rFried Chicken and Cheese Sandwich", "", "&7&oRestores &b&o" + "7.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("FRIED_CHICKEN_SANDWICH"), SlimefunItems.CHEESE, null, null, null, null, null, null, null},
+        14)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("NOODLES", "9174b34c549eed8bafe727618bab6821afcb1787b5decd1eecd6c213e7e7c6d", "&rNoodles", "", "&7&oRestores &b&o" + "5.0" + " &7&oHunger"),
+        new ItemStack[] {SlimefunItems.WHEAT_FLOUR, new ItemStack(Material.WATER_BUCKET), null, null, null, null, null, null, null},
+        10)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("BUTTERED_NOODLES", "9174b34c549eed8bafe727618bab6821afcb1787b5decd1eecd6c213e7e7c6d", "&rButtered Noodles", "", "&7&oRestores &b&o" + "6.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("NOODLES"), SlimefunItems.BUTTER, null, null, null, null, null, null, null},
+        12)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("MAC_N_CHEESE", "6bbecba5231805aaadda81d764b096eee62ed2e4cb447448544f5182b091f101", "&rMac n Cheese", "", "&7&oRestores &b&o" + "7.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("BUTTERED_NOODLES"), SlimefunItems.CHEESE, null, null, null, null, null, null, null},
+        14)
+        .register(plugin);

```

Comment:
Again, a ton of new random items, no explanation, no outlining, no docs, not a single notice. And out of scope.

Reply:
a) The work must carry prominent notices stating that you modified
    it, and giving a relevant date.  

Is having the source code on github for you to see my changes not enough?
How would you like me to convey these modifications to you. This is all new to me.
Ill gladly notify you when i modify the source code if i know how.

Reply:
This is all custom being used on my network only. I do not share this, besides that its forked as my own custom version.

Reply:
Well, in your pull request description you stated that you had fixed two bugs, nothing more than that.
When reviewing however, I instead found half of a McDonalds menu being added.

That's generally not a very good thing to do. Each Pull Request has a scope and you define this scope in the description. 
A large scope itself is already a problem since it is much harder to review a ton of changes at once, rather than individual changes individually. In this case, however the scope you defined ("Fixing 178 and 179") was exceeded by a huge margin.

Throwing random code at people and saying "here, you go figure it out" is not working out in the world of open-source. Documentation is key and so is honesty. That's why you not only tell people **what** you changed but also **why** and don't start adding unrelated things mid-way.

That puts a ton of work on the reviewers' side of things.
Remember: You are the one proposing these changes. And we are the ones who will end up having to maintain any code you submit for years to come.
We may agree with some of these changes but we might also disagree with other parts. Splitting it up into individual pull requests with a well-defined scope allows everyone to communicate about the individual features rather than rejecting all of them at once because we disagreed with one part of it.

I recommend you to really look into GitHub's docs about branches and how to use them. This way you can split up your commits among multiple branches and multiple pull requests, allowing you to stay in scope and allowing us to perform the reviews much quicker and more efficiently and effectively.